src/lib/Components/CellRenderer.tsx

```diff
@@ -134,7 +134,7 @@ export const CellRenderer: React.FC<CellRendererProps> = ({
             })}
         >
             {cellTemplate.render(cellToRender, isMobile ? isInEditMode : false, onCellChanged)}
-            {location.row.idx === 0 && location.column.resizable && <ResizeHandle />}
+            {(location.row.idx === 0 || cell.type === 'header')&& location.column.resizable && <ResizeHandle />}
```

Comment:
It's worth noting that enabling resizing for all header cells could affect users who rely on specific behavior for header cells in their projects (e.g., headers in the first column). To prevent unintended behavior, it would be better to create an optional flag, like `enableResizeOnAllHeaders`, to explicitly control this feature